android-agent/src/main/kotlin/io/opentelemetry/android/agent/OpenTelemetryRumInitializer.kt

```diff
@@ -72,13 +73,18 @@ object OpenTelemetryRumInitializer {
                 endpointBaseUrl,
                 endpointHeaders,
             ),
-        rumConfig: OtelRumConfig = OtelRumConfig(),
         sessionConfig: SessionConfig = SessionConfig.withDefaults(),
+        globalAttributes: (() -> Attributes)? = null,
         instrumentations: (InstrumentationConfiguration.() -> Unit)? = null,
     ): OpenTelemetryRum {
         instrumentations?.let { configure ->
             InstrumentationConfiguration().configure()
         }
+        val rumConfig = OtelRumConfig()
+        rumConfig.setDiskBufferingConfig(DiskBufferingConfig.create(enabled = true))
```

Comment:
At minimum, I think we need to expose a way to disable disk buffering and a way to turn on/off debugging for it (although that could be tied to a broader debug setting if one existed).

Reply:
Fair enough 👍 

I was thinking that it might be nice to reuse the same enabling/disabling config mechanism proposed for instrumentations [here](https://github.com/open-telemetry/opentelemetry-android/pull/1273) for diskbuffering as well. If you agree, then I'd like to hold this PR until that other one is merged, to do a rebase here and use the same config interface.